docs/development.md

```diff
@@ -73,8 +71,6 @@ If you have built llvm-project separately in the directory `$LLVM_INSTALL_DIR`,
 ```shell
 cmake -GNinja -Bbuild \
   -DCMAKE_BUILD_TYPE=Release \
-  -DCMAKE_C_COMPILER=clang \
-  -DCMAKE_CXX_COMPILER=clang++ \
```

Comment:
We can move this down as an addendum for folks who want to try clang.  Otherwise LGTM.